crates/header-translator/src/comment.rs

```diff
@@ -0,0 +1,126 @@
+//! Utilities for manipulating C/C++ comments.
```

Comment:
I copied this [file straight out of bindgen](https://github.com/rust-lang/rust-bindgen/blob/a9681090bf1d925b3da7444e75a64dfc3e46a827/bindgen/ir/comment.rs) and then modified it for some of the extra gross objective-c comments.